### DISCUSSION TITLE
Text content does not match server-rendered HTML.

### DISCUSSION THREAD

**Discussion by gaboesquivel** [vercel/next.js#68924] [App Router]
I want to display ***** as fallback instead of 0.0000 for client component that display crypto balances and I keep getting "Text content does not match server-rendered HTML. " error.   If I add suppressHydrationWarning to the component html is not updated. 

I'm using dynamic for now to get around it. Is there way to get the fallback with different format to work ?

**Comment by Juneezee** [comment]
You might want to try Solution 1 and Solution 2 suggested here: https://nextjs.org/docs/messages/react-hydration-error

**Reply by gaboesquivel** [reply]
Thanks @Juneezee, I double checked the tags in they are nested correctly.  it's actually just the text
If I use `0.0000` I dont get an error, only if I use `*****`
<img width="667" alt="image" src="https://github.com/user-attachments/assets/5de614a9-a683-454d-bbff-56ac3df9106d">
